otel-extensions/src/main/java/org/hypertrace/agent/otel/extensions/config/HypertraceConfig.java

```diff
@@ -143,9 +144,18 @@ static AgentConfig load(String filename) throws IOException {
     if (!builder.hasEndpoint()) {
       builder.setEndpoint(StringValue.newBuilder().setValue(DEFAULT_REPORTING_ENDPOINT).build());
     }
-    if (builder.getTraceReporterType().equals(Config.TraceReporterType.UNSPECIFIED)) {
+    if (!builder.hasMetricEndpoint()) {
```

Comment:
The test cases in `HypertraceConfigTest` should be expanded to include testing for these new properties

Reply:
done